composer.json

```
@@ -7,15 +7,17 @@
             "email" : "olivier.mansour@m6.fr"
         }
     ],
+    "config": {
+        "bin-dir": "bin/"
+    },
     "require" : {
-        "php" : "^7.0"
+        "php" : "^7.4|^8.0",
+        "symfony/symfony": "^2.7|^3.0|^4.0"
```

```suggestion
        "symfony/symfony": "^4.4"
```

(the others are not supported anymore)